pincer/utils/api_object.py

```diff
@@ -225,6 +225,14 @@ def __post_init__(self):
     def __factory__(cls: Generic[T], *args, **kwargs) -> T:
         return cls.from_dict(*args, **kwargs)
 
+    def __repr__(self):
+        attrs = ', '.join(
+            f"{k}={v!r}" for k, v in self.__dict__.items()
+            if not k.startswith('_') and v
```

Comment:
Should increase performance by like 20%
```suggestion
            if v and not k.startswith('_')
```